### PR TITLE
its all connected: read the description 👇🏼

### PR DESCRIPTION
**Pull request by kathinka**
- Update button text in Login page to "Sign in" instead of "Sign up".
- Connected points from form, travel, points and user.
- added useContext to be able to pass values among components in dash.
- made calculation for points.
- updated travel model to be able to pass on points to user and points db.
- updated user model.
- refactored travelform so we can get updates on points on the fly so the user can see the value of each travelchoice.
- and som nip and tucks.
- added ripple effect on ego button
- fixed links not working on landingpage
- fixed pointscard so it will update every 5 seconds
- fixed styling and global style
- fixed egobtn so it its easier to use overall

### CODE REVIEW

**Comment by kathinka** [comment]
DO NOT MERGE. only look don't touch. 😂 (conflict with Travelform - i had to restructured it)

File: frontend/src/pages/Login.jsx [hunk 1]
```diff
@@ -287,7 +287,7 @@ export const Login = () => {
 
             <ButtonAndSpinnerContainer>
               <EgoButton type="submit" disabled={loading}>
-                {loading ? "Loading..." : "Sign up"}
+                {loading ? "Loading..." : "Sign in"}
```
Comment (El1an3) [inline frontend/src/pages/Login.jsx]: Should be Log in.

**Review by El1an3 — APPROVED** [review]
Looks good to me...
Just lookie, no touchie 😂

**Review by vittoriamatteoli — APPROVED** [review]
Also just looking 😂 Smart adjustments📱

**Comment by kathinka** [comment]
Update button text in Login page to "Sign in" instead of "Sign up".
Connected points from form, travel, points and user.
added useContext to be able to pass values among components in dash.
made calculation for points.
updated travel model to be able to pass on points to user and points db.
updated user model.
refactored travelform so we can get updates on points on the fly so the user can see the value of each travelchoice.
and som nip and tucks.
added ripple effect on ego button
fixed links not working on landingpage
fixed pointscard so it will update every 5 seconds
fixed styling and global style
fixed egobtn so it its easier to use overall

**Pull request merged**